microsoft-edge/extensions-chromium/publish/reply-to-user-review.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Respond to user reviews about your extension
+description: Developers can view their Microsoft Edge extension’s reviews and ratings and respond to user reviews from the Partner Center dashboard.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 05/27/2021
+---
+# Respond to user reviews about your Microsoft Edge extension
+
+Microsoft recently launched a new feature enabling you to respond to user reviews received on the Microsoft Edge Add-ons product page.
+
+You can view your extension’s reviews and ratings and engage with user reviews from the Partner Center dashboard.
+
+Follow these steps to view your extension’s **Reviews** page:
```

Comment:
```suggestion
Follow these steps to view your extension's **Reviews** page:
```
https://review.docs.microsoft.com/en-us/help/contribute/markdown-reference?branch=main#apostrophes-and-quotation-marks